src/binding/julia/Access.cpp

```diff
@@ -0,0 +1,13 @@
+// IO/Access
```

Comment:
Let's add this everywhere:
```suggestion
/* This binds IO/Access
 *
 * File authors: Erik Schnetter
 * License: LGPL-3.0-or-later
 */
```